src/Mod/Image/ImageTools/_CommandImageScaling.py

```diff
@@ -35,7 +35,6 @@
     import FreeCADGui, FreeCAD, Part
     import math
     import pivy.coin as pvy
-    from PySide import QtCore, QtGui
```

Comment:
Did you remove this line? You shouldn't. Otherwise the QtCore and QtGui functions won't work.

My previous comment was that you don't need to explicitly load QtWidgets from PySide2. All QtWidgets are included in PySide.QtGui. See [PySide](https://wiki.freecadweb.org/PySide).

Reply:
Lines 33-34...
Just removed a duplicate.